jenkins/ci.suse.de/cloud-mkphyscloud-qa-rally-test.yaml

```diff
@@ -0,0 +1,101 @@
+- job:
+    name: cloud-mkphyscloud-qa-rally-test
+    node: cloud-mkphyscloud-gate-qa
+    description: |
+      Deploy vm on qa hw.
+      Setup rally envirment.
+      Run test
+      Mandatory parameter: hw_number
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    wrappers:
+      - build-name:
+          name: '#${BUILD_NUMBER} - ${scenario_name} - qa$hw_number - full'
+      - timestamps
+    publishers:
+      - mkphyscloud-qa-common-publishers
+
+    concurrent: true
+
+    parameters:
+      - string:
+          name: hw_number
+          default: "2"
+          description: Mandatory, number of the QA cloud server
+
+      - string:
+          name: image_name
+          default: jeos-64
```

Comment:
@Evalle in some cases image with jeos name already exist but I am not sure if it only xen specific or also kvm .I was thinking for our test always have new image,not one from tempest run.